qa/rpc-tests/wallet_unified_change.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+# Copyright (c) 2022 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://www.opensource.org/licenses/mit-license.php .
+
+from decimal import Decimal
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    NU5_BRANCH_ID,
+    assert_equal,
+    get_coinbase_address,
+    nuparams,
+    start_nodes,
+    wait_and_assert_operationid_status,
+)
+
+# Test wallet accounts behaviour
+class WalletUnifiedChangeTest(BitcoinTestFramework):
+    def __init__(self):
+        super().__init__()
+        self.num_nodes = 4
+
+    def setup_nodes(self):
+        return start_nodes(self.num_nodes, self.options.tmpdir, [[
+            nuparams(NU5_BRANCH_ID, 201),
+        ]] * self.num_nodes)
+
+    def run_test(self):
+        # Activate Nu5
+        self.sync_all()
+        self.nodes[0].generate(1)
+        self.sync_all()
+
+        account0 = self.nodes[0].z_getnewaccount()['account']
+        ua0_sapling = self.nodes[0].z_getaddressforaccount(account0, ['sapling'])['address']
+        ua0_orchard = self.nodes[0].z_getaddressforaccount(account0, ['orchard'])['address']
+
+        account1 = self.nodes[1].z_getnewaccount()['account']
+        ua1_sapling = self.nodes[1].z_getaddressforaccount(account1, ['sapling'])['address']
+        ua1 = self.nodes[1].z_getaddressforaccount(account1)['address']
+
+        # Fund both of ua0_sapling and ua0_orchard
+        recipients = [{'address': ua0_sapling, 'amount': 10}]
+        opid = self.nodes[0].z_sendmany(get_coinbase_address(self.nodes[0]), recipients, 1, 0, 'AllowRevealedSenders')
+        wait_and_assert_operationid_status(self.nodes[0], opid)
+
+        recipients = [{'address': ua0_orchard, 'amount': 10}]
+        opid = self.nodes[0].z_sendmany(get_coinbase_address(self.nodes[0]), recipients, 1, 0, 'AllowRevealedSenders')
+        wait_and_assert_operationid_status(self.nodes[0], opid)
+
+        self.sync_all()
+        self.nodes[0].generate(1)
+        self.sync_all()
+
+        assert_equal(
+                {'pools': {'sapling': {'valueZat': 1000000000}, 'orchard': {'valueZat': 1000000000}}, 'minimum_confirmations': 1},
+                self.nodes[0].z_getbalanceforaccount(account0))
+
+        # Send both amounts to ua1 in fully-shielded transactions. This will result
+        # in account1 having both a Sapling and an Orchard balances.
```

Comment:
```suggestion
        # in account1 having both Sapling and Orchard balances.
```

Reply:
Will fix in the follow-up PR that makes all policy violation checking async.